spellbook/models/uniswap/optimism/uniswap_optimism_schema.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+
+models:
+  - name: uniswap_optimism_pools
+    meta:
+      blockchain: optimism
+      project: uniswap
+      contributors: MSilb7, chuxinh
+    config:
+      tags: ['optimism','uniswap','pools']
+    description: >
+        A canonical table containing all Uniswap pools created on Optimism.
+        tests:
```

Comment:
@chuxinh it doesn't seem like this test is triggering anything. I ran the build command and no tests ran. I think you need to move it under a column. Is pool the column you'd like to check for uniqueness?

```
models:
  - name: uniswap_optimism_pools
    meta:
      blockchain: optimism
      project: uniswap
      contributors: MSilb7, chuxinh
    config:
      tags: ['optimism','uniswap','pools']
    description: >
        A canonical table containing all Uniswap pools created on Optimism.
    columns:
      - &pool
        name: pool
        description: "Uniswap liquidity pool contract address"
        tests:
          - unique

Reply:
Oh yea, thanks for this! 
Have it updated